cmd/hsd/server.go

```diff
@@ -500,7 +500,7 @@ func (srv *Server) loadModules() error {
 	if strings.Contains(srv.config.Siad.Modules, "c") {
 		i++
 		fmt.Printf("(%d/%d) Loading consensus...\n", i, len(srv.config.Siad.Modules))
-		cs, err = consensus.New(g, !srv.config.Siad.NoBootstrap, filepath.Join(srv.config.Siad.SiaDir, modules.ConsensusDir))
+		cs, err = consensus.NewConsensus(g, !srv.config.Siad.NoBootstrap, filepath.Join(srv.config.Siad.SiaDir, modules.ConsensusDir), srv.config.Siad.Spv)
```

Comment:
I prefer we stick with consensus.New here, as all the other modules in the codebase use that pattern.